### PR TITLE
Add TrOCR resources

### PR DESCRIPTION
**Pull request by huangperry**
# What does this PR do?

Adds resources of OpenAI GPT according to https://github.com/huggingface/transformers/issues/20055

Fixes #20055 (partially)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@stevhliu

### CODE REVIEW

File: docs/source/en/model_doc/trocr.mdx [hunk 1]
```diff
@@ -50,6 +50,31 @@ Tips:
...
+- A blog post on how to [Document AI](https://github.com/philschmid/document-ai-transformers) with TrOCR. 🌎
+- A notebook on how to [finetune TrOCR on IAM Handwriting Database using Seq2SeqTrainer](https://colab.research.google.com/github/NielsRogge/Transformers-Tutorials/blob/master/TrOCR/Fine_tune_TrOCR_on_IAM_Handwriting_Database_using_Seq2SeqTrainer.ipynb). 🌎
+- A notebook on [inference with TrOCR](https://colab.research.google.com/github/NielsRogge/Transformers-Tutorials/blob/master/TrOCR/Inference_with_TrOCR_%2B_Gradio_demo.ipynb) and Gradio demo. 🌎
+- A notebook on [finetune TrOCR on the IAM Handwriting Database](https://colab.research.google.com/github/NielsRogge/Transformers-Tutorials/blob/master/TrOCR/Fine_tune_TrOCR_on_IAM_Handwriting_Database_using_native_PyTorch.ipynb) using native PyTorch. 🌎
+- A notebook on [evaluating TrOCR on the IAM test set](https://colab.research.google.com/github/NielsRogge/Transformers-Tutorials/blob/master/TrOCR/Evaluating_TrOCR_base_handwritten_on_the_IAM_test_set.ipynb). 🌎
+
+<PipelineTag pipeline="token-classification"/>
+
+- [Token classification](https://huggingface.co/course/chapter7/2?fw=pt) chapter of the 🤗 Hugging Face Course.
```
Comment (stevhliu) [inline docs/source/en/model_doc/trocr.mdx:68]: I don't think token classification is relevant here, it's more of an image understanding/text generation task.

**Review by stevhliu — APPROVED** [review]
Thanks for your contribution; I like the link to the interactive demo! I just left a few comments, otherwise looks good 🤗

File: docs/source/en/model_doc/trocr.mdx [hunk 2]
```diff
@@ -50,6 +50,31 @@ Tips:
   information, see the [official models](https://huggingface.co/models?other=trocr>).
 - TrOCR is always used within the [VisionEncoderDecoder](vision-encoder-decoder) framework.
 
+## Resources
+
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with TrOCR. If you're interested in submitting a resource to be included here, please feel free to open a Pull Request and we'll review it! The resource should ideally demonstrate something new instead of duplicating an existing resource.
+
+<PipelineTag pipeline="text-classification"/>
+
+- A blog post on [Accelerating Document AI](https://huggingface.co/blog/document-ai) with TrOCR. 🌎
```
[the remark this answers is not in the document]
Comment (huangperry) [reply docs/source/en/model_doc/trocr.mdx:59]: Removed 🌎s.

**Review by sgugger — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**